pages/featured.jsx

```diff
@@ -1,10 +1,7 @@
 import React from 'react';
-import ProjectList from '../components/project-list/project-list.jsx';
+import ProjectLoader from '../components/project-loader/project-loader.jsx';
+
+export default function () {
+  return <div><ProjectLoader params={{featured: `True`}} /></div>;
```

Comment:
why is there a `<div>` around this? Shouldn't the ProjectLoader return its own content wrapper with associated className?